lib/rubocop/cop/lint/assignment_in_condition.rb

```diff
@@ -24,7 +24,10 @@ module Lint
       class AssignmentInCondition < Cop
         include SafeAssignment
 
-        MSG = 'Assignment in condition - you probably meant to use `==`.'.freeze
+        MSG = [
```

Comment:
There's an option controlling whether this is ok or not, so you can just edit the message directly.